src/azul/queues.py

```diff
@@ -213,7 +213,8 @@ def wait_to_stabilize(self) -> int:
         """
         sleep_time = 10
         queues = self.get_queues(config.work_queue_names)
-        total_lengths = deque(maxlen=12)
+        # A generous 10 minutes to accommodate transient stalls
+        total_lengths = deque(maxlen=10 * 60 // sleep_time)
```

Comment:
With `//`, if sleep time is 14 the resulting time window would be 588. We want at least 10 min.